deploy/tomcat/context.xml

```diff
@@ -0,0 +1,43 @@
+<!--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2021 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<!DOCTYPE doc>
+<!--
+    CU Customizations:
+
+    * Added settings for the "path" and "reloadable" attributes, as well as a DOCTYPE declaration.
+    * Added "CookieProcessor" settings to fix some Shibboleth integration issues.
+    * Backported FINP-7556 resource cache fix from the 2021-05-13 patch.
+    * Backported FINP-7585 context.xml bundling fix from the 2021-05-27 patch (which moved this file
+          to a new location and required adjusting the POM's maven-war-plugin configuration).
+ -->
```

Comment:
The comment above talks about adding a DOCTYPE declaration (which was in the file that was removed) but I do not see it in the XML code below.  Should the comment be fixed or does the XML need adjusting?

Reply:
The DOCTYPE part is still in this file, though it's above the initial comment instead of below it. Sorry for the confusion.

If you want me to move that DOCTYPE declaration below the comment, please let me know.

Reply:
If it is not too much trouble, I think having the comment firs(as it talks about all the changes) followed by the changes would keep this closer to the existing mod being moved and prevent others from missing that single statement in the future.

Reply:
Moved its position as requested.